source/dofs/dof_handler_policy.cc

```diff
@@ -134,6 +134,146 @@ namespace internal
                            2*MultithreadInfo::n_threads(),
                            /* chunk_size = */ 32);
         }
+
+
+        typedef
+        std::vector<std::pair<unsigned int, unsigned int> > DoFIdentities;
+
+
+        /**
+         * Make sure that the given @p
```

Comment:
OK. I can't do this as part of the current PR because it would conflict with commits I already have on top of the work here, but I've rewrapped these and other comments in a later commit that will come as part of the next PR: https://github.com/bangerth/dealii/commit/ccb32a3d7557db13d329d41633a98240dfba4373